### PR TITLE
[Docs] Mutes Cog Guide

### PR DESCRIPTION
**Pull request by Kreusada**
### Type

- [ ] Bugfix
- [ ] Enhancement
- [ ] New feature
- [x] Documentation

### Description of the changes

The Mutes cog guide. (#1734)

### CODE REVIEW

**Review by Jackenmen — COMMENTED** [review]
This PR should update the `.github/labeler.yml` file similarly to how it's done for existing cog guides:
https://github.com/Cog-Creators/Red-DiscordBot/blob/V3/develop/.github/labeler.yml#L1-L5

```yaml
"Category: Mutes Cog":
  # Source
  - redbot/cogs/mutes/*
  # Docs
  - docs/cog_guides/mutes.rst
```

File: .github/labeler.yml [hunk 1]
```diff
@@ -74,6 +74,7 @@
   - redbot/cogs/modlog/*
 "Category: Mutes Cog":
   - redbot/cogs/mutes/*
+  - docs/cog_guides/mutes.rst
```
Comment (Jackenmen) [inline .github/labeler.yml]: For consistency with the other entries in the labeler file (the one in the branch this PR is created from is outdated so you can't really see that's the case), there should be appropriate comments:
```suggestion
"Category: Mutes Cog":
  # Source
  - redbot/cogs/mutes/*
  # Docs
  - docs/cog_guides/mutes.rst
```

File: docs/cog_guides/mutes.rst [hunk 2]
```diff
@@ -0,0 +1,409 @@
...
+
+    [p]muteset forcerole <true_or_false>
+
+**Description**
+
+Whether or not to force role only mutes on the bot.
+
+**Arguments**
+
+* ``<true_or_false>``: Whether or not to enable/disable this setting, most provide ``true`` or ``false``.
```
Comment (Flame442) [inline docs/cog_guides/mutes.rst]:
```suggestion
* ``<true_or_false>``: Whether to enable or disable this setting, must provide ``true`` or ``false``.
```

File: docs/cog_guides/mutes.rst [hunk 3]
```diff
@@ -0,0 +1,409 @@
...
+**Description**
+
+Sets the role to be applied when muting a user.
+
+If no role is setup the bot will attempt to mute a user by setting
+channel overwrites in all channels to prevent the user from sending messages.
+
+.. Note:: If no role is setup a user may be able to leave the server
+
+and rejoin no longer being muted.
```
Comment (Flame442) [inline docs/cog_guides/mutes.rst]: image
The syntax on this is broken

File: docs/cog_guides/mutes.rst [hunk 4]
```diff
@@ -0,0 +1,409 @@
...
+
+    [p]muteset senddm <true_or_false>
+
+**Description**
+
+Set whether mute notifications should be sent to users in DMs.
+
+**Arguments**
+
+* ``<true_or_false>``: Whether or not to enable/disable this setting, most provide ``true`` or ``false``.
```
Comment (Flame442) [inline docs/cog_guides/mutes.rst]:
```suggestion
* ``<true_or_false>``: Whether to enable or disable this setting, must provide ``true`` or ``false``.
```

File: docs/cog_guides/mutes.rst [hunk 5]
```diff
@@ -0,0 +1,409 @@
...
+
+    [p]muteset showmoderator <true_or_false>
+
+**Description**
+
+Decide whether the name of the moderator muting a user should be included in the DM to that user.
+
+**Arguments**
+
+* ``<true_or_false>``: Whether or not to enable/disable this setting, most provide ``true`` or ``false``.
```
Comment (Flame442) [inline docs/cog_guides/mutes.rst]:
```suggestion
* ``<true_or_false>``: Whether to enable or disable this setting, must provide ``true`` or ``false``.
```

**Review by Flame442 — CHANGES_REQUESTED** [review]
A few little things, but otherwise good.

**Comment by Kreusada** [comment]
Thanks for reviewing, I've made the requested changes.

**Review by Flame442 — APPROVED** [review]
Thanks for making those changes, all set to go now!

**Pull request merged**